schema/dereferenced-release-schema.json

```diff
@@ -5304,7 +5304,7 @@
           },
           "documents": {
             "title": "Documents",
-            "description": "All documents and attachments related to the award, including any notices.",
+            "description": "Documents related to the award stage (for example notices, bids, evaluation reports).",
```

Comment:
For award, contract and implementation, the documents relate to the specific award, contract, and contract implementation – not the stage as a whole.

Reply:
Makes sense. I wasn't sure whether implementation should be "implementation", "the implementation" or "implementation of the contract". I've tentatively included it with the last option.

Reply:
Added in https://github.com/open-contracting/standard/pull/1229/commits/c351ca4cbb9df2edaaebda33100ede21ee1069db